shoplist/src/main/java/com/netguru/codereview/ui/model/ShopList.kt

```diff
@@ -0,0 +1,11 @@
+package com.netguru.codereview.ui.model
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+
+class ShopList(
```

Comment:
- [ ] Make ShopList a Data class. Since it is a holding class. Kotlin data class comes with functionality relevant.   [Click here](https://kotlinlang.org/docs/data-classes.html) for more info